internal/pkg/install/install.go

```diff
@@ -38,6 +39,8 @@ func RunInstallerContainer(r runtime.Runtime, opts ...Option) error {
 		return err
 	}
 
+	log.Printf("pulling installer container image: %q", r.Config().Machine().Install().Image())
```

Comment:
I added this as when image pull hangs on retries, it's hard to understand why boot hangs